dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/GeminiTool.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.GoogleVertexAI;
+
+/// <summary>
+/// A Tool is a piece of code that enables the system to interact with external systems to perform an action,
+/// or set of actions, outside of knowledge and scope of the model.
+/// </summary>
+internal sealed class GeminiTool
```

Comment:
All internal classes, that are within Core, can be in the `.Core` namespace. 

Make sense to keep in the `GoogleVertexAI` the public available ones.